File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/python/clases_orientacion_objetos/5_clase_de_agosto_09/sobreescribir_metodo.py
class Humano:
    nombre = ""
    sexo = ""
    nacionalidad = ""
    
    def __init__(self,sexo,nacionalidad,nombre):
        self.sexo = sexo
        self.nacionalidad = nacionalidad
        self.nombre = nombre
            
    def toString(self):
        print("Se ha creado un nuevo un humano de genero "+self.sexo+", con el nombre "+self.nombre+" y de nacionalidad "+self.nacionalidad)
        
        
        
nuevo1 = Humano("hombre","keiner lindarte","colombiano")
print(nuevo1.toString())

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/java/clases_orientacion_objetos/clase_5_de_agosto_09/sobreescribir_metodo.java
package ADSO2928707.portafolio_aprendices.keiner_lindarte_aro.instructor_henry.mis_programas.practicas.java.clases_orientacion_objetos.clase_5_de_agosto_09;

public class sobreescribir_metodo {
    
    public static void main(String[] args) {
        
        Humano nuevo1 = new Humano("hombre", "colombiano", "keiner lindarte");
        System.out.println(nuevo1.toString());
    }
}

class Humano {

    String nombre = "";
    String sexo = "";
    String nacionalidad = "";

    Humano(String sexo, String nacionalidad, String nombre){
        this.sexo = sexo;
        this.nacionalidad = nacionalidad;
        this.nombre = nombre;
    }

    public String toString(){
        return "Se ha creado un nuevo un humano de genero "+sexo+", con el nombre :"+nombre+" y de nacionalidad "+nacionalidad;
    }
    

}